specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -0,0 +1,1317 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-06-25",
```

Comment:
>2020-06-25 [](start = 16, length = 10)

please make the first commit the old api-versions content in the new path then start making changes to make these easier to review in the future